spec/classes/init_spec.rb

```diff
@@ -514,7 +514,7 @@
           }
         end
 
-        it { should_not contain_file('ntp_conf').with_content(/^\s*statsdir/) }
+        it { should_not contain_file('ntp_conf').with_content(/statsdir \/path\/to\/statsdir.statistics loopstats peerstats clockstats.filegen loopstats file loopstats type day enable.filegen peerstats file peerstats type day enable.filegen clockstats file clockstats type day enable/m) }
```

Comment:
This should have the `^` at the beginning still. Can you do newlines in the regex?